packages/atomic/src/components/common/iconButton.tsx

```diff
@@ -0,0 +1,33 @@
+import {FunctionalComponent, h, VNode} from '@stencil/core';
+import {Button, ButtonProps} from './button';
+
+export interface IconButtonProps extends ButtonProps {
+  badge?: VNode;
+  buttonRef?: (el?: HTMLButtonElement) => void;
+  icon: string;
+}
+
+export const IconButton: FunctionalComponent<IconButtonProps> = (props) => {
+  return (
+    <div class="relative" part="container">
+      <Button
+        {...props}
+        class="p-3 relative"
+        part="button"
```

Comment:
I think all parts in here should become required props, like either:
```ts
export interface IconButtonProps extends ButtonProps {
  badge?: VNode;
  buttonRef?: (el?: HTMLButtonElement) => void;
  icon: string;
  buttonPart: string;
  iconPart: string;
  badgePart: string;
}
```
or
```ts
export interface IconButtonProps extends ButtonProps {
  badge?: VNode;
  buttonRef?: (el?: HTMLButtonElement) => void;
  icon: string;
  partPrefix: string;
}
```

The reason is that all these components have styling, and therefore require parts to be customized. But also, a name like "button" or "icon" may not be relevant to a particular component, and a component may render multiple icon buttons.